src/opam.nix

```diff
@@ -562,6 +562,19 @@ in rec {
 
   # remove non-dune dependant packages
   opamRepositoryFiltered =
+    # TODO is there a way to do this outside a derivation?
+    let parseOpam = pkgPath:
```

Comment:
Why not just use `importOpam`?

Reply:
Purely aesthetics. It's hard to judge progress when building  ~24k derivations with the same name 😄 I can add this to `importOpam` if you think it's nice!